.github/workflows/main.yml

```diff
@@ -25,12 +23,16 @@ jobs:
   appimage:
     needs: test
     uses: viamrobotics/rdk/.github/workflows/appimage.yml@main
+    with:
```

Comment:
is there a way to set this once in the overall workflow?

Reply:
I tried with env variables, but they don't pass through. Only way I could find was to use inputs to each workflow like this (same as we have to do with secrets.)

Reply:
ah no worries then!